config/defaults.ini.php

```diff
@@ -0,0 +1,457 @@
+<?php return <<<INI
+
+; This file declares the default values for all the config values of Friendica.
+; Please don't edit this file as its content may change in the upcoming versions.
+
+[jabber]
+
+; debug (Boolean)
+; Enable debug level for the jabber account synchronisation.
+debug = false
+
+; lockpath (Path)
+; Must be writable by the ejabberd process. if set then it will prevent the running of multiple processes.
+lockpath =
+
+[database]
+
+; host (String)
+; Hostname or IP address of the database server.
+; Can contain the port number with the syntax "hostname:port".
+hostname =
+
+; user (String)
+; Database user name. Please don't use "root".
+username =
+
+; pass (String)
+; Database user password. Please don't use empty passwords.
+password =
+
+; base (String)
+; Database name.
+database =
+
+; charset (String)
+; Database connexion charset. Changing this value will likely corrupt special characters.
+charset = utf8mb4
+
+[config]
+
+; admin_email (String)
```

Comment:
What is the new format for multiple admins? Currently it is a comma separated list of email addresses.

Reply:
It stays the same.